homework/20/politicion-beans/main/java/com/example/Politition.java

```diff
@@ -0,0 +1,6 @@
+package com.example;
+
+public abstract class Politition {

```

Comment:
How did you choose whether to use abstract class or interface? 🤔

Reply:
The child classes have similar functionality and because like this my code would have a better structure.